charts/thoras/templates/dashboard/deployment.yaml

```diff
@@ -52,11 +62,6 @@ spec:
           {{- end }}
           - name: "MODEL_IMAGE"
             value: {{ .Values.imageCredentials.registry }}/thoras-forecast:{{ default .Values.thorasVersion .Values.thorasForecast.imageTag }}
-          - name: SLACK_WEBHOOK_URL
```

Comment:
I moved this so the env field we're testing is in the same array index across all deployments . This let's us to have one helm unit test instead of many and the env var ordering shouldn't matter